app/src/main/res/css/modules/compact_user_card.sass

```diff
@@ -51,8 +52,8 @@
       cursor: pointer
   .logout
     position: absolute
-    bottom: 5px
-    left: 10px
+    bottom: 10px
```

Comment:
Done. Initially I used `px` instead of `rem` because the previous commit also used it.
I suppose most of the units should be using `rem` here, right? I'm on 150% dpi myself.